cypress/plugins/index.js

```diff
@@ -40,6 +40,14 @@ module.exports = (on, config) => {
 
   on('before:browser:launch', (browser = {}, launchOptions) => {
     prepareAudit(launchOptions)
+
+    launchOptions.preferences.default['default.disable_3d_apis'] = false
+
+    launchOptions.args.push('--enable-features=VaapiVideoDecoder')
+    launchOptions.args.push('--disable-gpu')
+    launchOptions.args.push('--use-gl=egl')
```

Comment:
Comments for future maintainers would be a nice touch here as the purpose of these args is somewhat mysterious.